resources/timerbar.ts

```diff
@@ -485,7 +485,7 @@ export default class TimerBar extends HTMLElement {
   }
 
   // Apply all styles from an object where keys are CSS properties
-  applyStyles(styles: { [s: string]: string}): void {
+  applyStyles(styles: { [s: string]: string }): void {
```

Comment:
Thanks! I wonder if there's eslint for this.

Reply:
I think it is [@typescript-eslint/object-curly-spacing](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/object-curly-spacing.md)